src/main/java/uk/gov/register/presentation/representations/TurtleWriter.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.register.presentation.representations;
+
+import com.google.common.base.Throwables;
+import uk.gov.register.presentation.Entry;
+import uk.gov.register.presentation.mapper.JsonObjectMapper;
+import uk.gov.register.presentation.view.AbstractView;
+import uk.gov.register.presentation.view.ListResultView;
+import uk.gov.register.presentation.view.SingleResultView;
+
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.UriBuilder;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Produces(ExtraMediaType.TEXT_TTL)
+public class TurtleWriter extends RepresentationWriter<AbstractView> {
+
+
+    @Override
+    public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
+        return type.isAssignableFrom(SingleResultView.class) || type.isAssignableFrom(ListResultView.class);
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void writeTo(AbstractView view, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType, MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream) throws IOException, WebApplicationException {
+        List<Entry> entries = getEntries(view.get());
+
+        List<String> fields = entryFields(JsonObjectMapper.convert(entries.get(0).getContent(), Map.class));
+        for (Entry entry : entries) {
+            writeRow(entry, fields, entityStream);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    private List<Entry> getEntries(Object o) {
+        if (o instanceof List) {
```

Comment:
Would it be nicer to do the instanceof checks on `view` rather than on the result of `view.get()`?